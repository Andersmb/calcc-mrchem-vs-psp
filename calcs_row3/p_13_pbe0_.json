{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.8773858368179578
          ],
          "radius": 1.23
        },
        {
          "center": [
            0.0,
            0.0,
            1.9401956131820421
          ],
          "radius": 0.75
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "p",
          "xyz": [
            0.0,
            0.0,
            -0.8773858368179578
          ]
        },
        {
          "atom": "n",
          "xyz": [
            0.0,
            0.0,
            1.9401956131820421
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "p_13_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        1.2588466489141057e-16
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.1329700775540434,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -7.511561961903024e-08,
            3.967380722340114e-09,
            -1.132970077554041
          ],
          "vector_el": [
            -7.523817455543521e-08,
            3.8447968214283524e-09,
            -1.5535518175630092
          ],
          "vector_nuc": [
            1.2255493640496965e-10,
            1.22583900911762e-10,
            0.4205817400089682
          ]
        }
      },
      "geometry": [
        {
          "symbol": "P",
          "xyz": [
            0.0,
            0.0,
            -0.8773858368179578
          ]
        },
        {
          "symbol": "N",
          "xyz": [
            0.0,
            0.0,
            1.9401956131820421
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -77.25379836731679,
          -14.422504496342988,
          -6.656255964971064,
          -4.7917830204633365,
          -4.791783015788175,
          -4.786890790809991,
          -0.8767738033479033,
          -0.5126767614890996,
          -0.34978745138387113,
          -0.34978744068522205,
          -0.3424334361053783,
          -77.25379837057876,
          -14.422504484810224,
          -6.656255973324312,
          -4.791783024906166,
          -4.791783018096772,
          -4.786890801579496,
          -0.8767738282166584,
          -0.5126767690052886,
          -0.34978743516806077,
          -0.3497874249952791,
          -0.3424334620346996
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -230.26894914141954
      },
      "scf_energy": {
        "E_ee": 216.74621370983743,
        "E_eext": 0.0,
        "E_el": -433.180404954235,
        "E_en": -1015.2652532442959,
        "E_kin": 395.1849788743582,
        "E_next": 0.0,
        "E_nn": 37.266003437096735,
        "E_nuc": 37.266003437096735,
        "E_tot": -395.91440151713823,
        "E_x": -7.276441410664222,
        "E_xc": -22.56990288347051,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 216.6672431429725,
        "E_eext": 0.0,
        "E_el": -430.9101970300447,
        "E_en": -1010.2883542002018,
        "E_kin": 392.6611259262602,
        "E_next": 0.0,
        "E_nn": 37.266003437096735,
        "E_nuc": 37.266003437096735,
        "E_tot": -393.64419359294794,
        "E_x": -7.3066351864512935,
        "E_xc": -22.64357671262432,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 214.16797221648008,
              "E_eext": 0.0,
              "E_el": -433.03435680199135,
              "E_en": -1007.9767021892694,
              "E_kin": 390.3378437954571,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.7683533648946,
              "E_x": -7.2085071947554065,
              "E_xc": -22.354963429903655,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.7683533648946,
            "energy_update": 2.1241597719466654,
            "mo_residual": 0.5152129828059598,
            "wall_time": 98.434849859
          },
          {
            "energy_terms": {
              "E_ee": 216.54912893959283,
              "E_eext": 0.0,
              "E_el": -433.15281783838253,
              "E_en": -1013.4194731144362,
              "E_kin": 393.54742317814623,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.8868144012858,
              "E_x": -7.2728314200371775,
              "E_xc": -22.557065421648247,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.8868144012858,
            "energy_update": 0.11846103639118155,
            "mo_residual": 0.272275482309015,
            "wall_time": 70.899918463
          },
          {
            "energy_terms": {
              "E_ee": 216.29248667172294,
              "E_eext": 0.0,
              "E_el": -433.1707945409285,
              "E_en": -1013.9598003477793,
              "E_kin": 394.29517405793626,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.90479110383177,
              "E_x": -7.264428846696309,
              "E_xc": -22.534226076112116,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.90479110383177,
            "energy_update": 0.017976702545979606,
            "mo_residual": 0.1185899861267027,
            "wall_time": 62.013166499
          },
          {
            "energy_terms": {
              "E_ee": 216.5655475014209,
              "E_eext": 0.0,
              "E_el": -433.17639516494825,
              "E_en": -1014.6056615583627,
              "E_kin": 394.69340187018275,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.9103917278515,
              "E_x": -7.272047765321016,
              "E_xc": -22.557635212868078,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.9103917278515,
            "energy_update": 0.005600624019734823,
            "mo_residual": 0.053825439025201885,
            "wall_time": 60.720386227
          },
          {
            "energy_terms": {
              "E_ee": 216.77425704555395,
              "E_eext": 0.0,
              "E_el": -433.17964734523474,
              "E_en": -1015.4101122063014,
              "E_kin": 395.3081573665582,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.913643908138,
              "E_x": -7.277343813730299,
              "E_xc": -22.574605737315174,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.913643908138,
            "energy_update": 0.0032521802864948768,
            "mo_residual": 0.040639448068968784,
            "wall_time": 61.733643812
          },
          {
            "energy_terms": {
              "E_ee": 216.7714692375897,
              "E_eext": 0.0,
              "E_el": -433.1800849357808,
              "E_en": -1015.4442265171217,
              "E_kin": 395.3435107195298,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.9140814986841,
              "E_x": -7.277155302593953,
              "E_xc": -22.573683073184743,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.9140814986841,
            "energy_update": 0.0004375905460847207,
            "mo_residual": 0.013962882998760973,
            "wall_time": 61.914335043
          },
          {
            "energy_terms": {
              "E_ee": 216.7487282128524,
              "E_eext": 0.0,
              "E_el": -433.1803198611751,
              "E_en": -1015.2954761128531,
              "E_kin": 395.2133681561489,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.9143164240784,
              "E_x": -7.276433357539978,
              "E_xc": -22.570506759783257,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.9143164240784,
            "energy_update": 0.00023492539429526005,
            "mo_residual": 0.01024253829722575,
            "wall_time": 63.719127442
          },
          {
            "energy_terms": {
              "E_ee": 216.74727117872874,
              "E_eext": 0.0,
              "E_el": -433.1803777783456,
              "E_en": -1015.2388047220937,
              "E_kin": 395.15714921374894,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.91437434124884,
              "E_x": -7.276343397217711,
              "E_xc": -22.569650051511857,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.91437434124884,
            "energy_update": 5.791717046577105e-05,
            "mo_residual": 0.007595513661829171,
            "wall_time": 63.406416789
          },
          {
            "energy_terms": {
              "E_ee": 216.74643130443295,
              "E_eext": 0.0,
              "E_el": -433.1803954560633,
              "E_en": -1015.2427060197725,
              "E_kin": 395.16188352760463,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.91439201896657,
              "E_x": -7.276383000375909,
              "E_xc": -22.569621267952378,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.91439201896657,
            "energy_update": 1.7677717721653607e-05,
            "mo_residual": 0.004108002369694279,
            "wall_time": 65.373403437
          },
          {
            "energy_terms": {
              "E_ee": 216.74377721245227,
              "E_eext": 0.0,
              "E_el": -433.1804016715463,
              "E_en": -1015.2501456160818,
              "E_kin": 395.17194155551914,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.91439823444955,
              "E_x": -7.2763765141655945,
              "E_xc": -22.569598309270333,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.91439823444955,
            "energy_update": 6.215482983407128e-06,
            "mo_residual": 0.0020162548026873787,
            "wall_time": 66.580757753
          },
          {
            "energy_terms": {
              "E_ee": 216.7441183905844,
              "E_eext": 0.0,
              "E_el": -433.18040417435475,
              "E_en": -1015.261969331197,
              "E_kin": 395.183649081761,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.914400737258,
              "E_x": -7.276421507962712,
              "E_xc": -22.569780807540496,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.914400737258,
            "energy_update": 2.5028084564837627e-06,
            "mo_residual": 0.0013184884277922227,
            "wall_time": 66.232151217
          },
          {
            "energy_terms": {
              "E_ee": 216.74575101833764,
              "E_eext": 0.0,
              "E_el": -433.18040476113714,
              "E_en": -1015.2679394890794,
              "E_kin": 395.18817034108946,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.9144013240404,
              "E_x": -7.27645683397502,
              "E_xc": -22.56992979750973,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.9144013240404,
            "energy_update": 5.867823915650661e-07,
            "mo_residual": 0.0005473289206770712,
            "wall_time": 68.399983831
          },
          {
            "energy_terms": {
              "E_ee": 216.74633583048296,
              "E_eext": 0.0,
              "E_el": -433.18040491026755,
              "E_en": -1015.2675251128503,
              "E_kin": 395.1871832665854,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.9144014731708,
              "E_x": -7.276455293532612,
              "E_xc": -22.569943600953042,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.9144014731708,
            "energy_update": 1.4913041468389565e-07,
            "mo_residual": 0.0002839425746261397,
            "wall_time": 67.781822762
          },
          {
            "energy_terms": {
              "E_ee": 216.74621370983743,
              "E_eext": 0.0,
              "E_el": -433.180404954235,
              "E_en": -1015.2652532442959,
              "E_kin": 395.1849788743582,
              "E_next": 0.0,
              "E_nn": 37.266003437096735,
              "E_nuc": 37.266003437096735,
              "E_tot": -395.91440151713823,
              "E_x": -7.276441410664222,
              "E_xc": -22.56990288347051,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -395.91440151713823,
            "energy_update": 4.3967418150714366e-08,
            "mo_residual": 0.00011989836273513816,
            "wall_time": 78.090770818
          }
        ],
        "wall_time": 955.336459786
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
