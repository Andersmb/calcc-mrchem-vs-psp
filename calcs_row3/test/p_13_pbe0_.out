                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -0.877386
    1       N            :        0.000000        0.000000        1.940196
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              11
 Beta electrons          :                                              11
 Total electrons         :                                              22
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999957718e-01
    1    1    a          :                              9.999999368611e-01
    2    1    a          :                              9.999999999838e-01
    3    1    a          :                              9.999999999453e-01
    4    1    a          :                              9.999999997459e-01
    5    1    a          :                              9.999999997459e-01
    6    1    a          :                              9.999999989276e-01
    7    1    a          :                              1.000000008169e+00
    8    1    a          :                              9.999999766779e-01
    9    1    a          :                              9.999999766693e-01
   10    1    a          :                              9.999999964307e-01
   11    1    b          :                              1.000000000380e+00
   12    1    b          :                              9.999999699010e-01
   13    1    b          :                              9.999999991993e-01
   14    1    b          :                              1.000000004232e+00
   15    1    b          :                              9.999999997459e-01
   16    1    b          :                              9.999999997459e-01
   17    1    b          :                              1.000000045341e+00
   18    1    b          :                              9.999999712535e-01
   19    1    b          :                              9.999999756460e-01
   20    1    b          :                              9.999999776592e-01
   21    1    b          :                              9.999999964307e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                392.661125926260
 E-N energy              :            (au)              -1010.288354200202
 Coulomb energy          :            (au)                216.667243142972
 Exchange energy         :            (au)                 -7.306635186451
 X-C energy              :            (au)                -22.643576712624
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 37.266003437097
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -430.910197030045
 Nuclear energy          :            (au)                 37.266003437097
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.936441935929e+02
                         :      (kcal/mol)             -2.470154588964e+05
                         :        (kJ/mol)             -1.033512680693e+06
                         :            (eV)             -1.071160372455e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.931071978082
    1    1    a          :            (au)                -13.834606899756
    2    1    a          :            (au)                 -6.781646647328
    3    1    a          :            (au)                 -4.922247636875
    4    1    a          :            (au)                 -4.922240933311
    5    1    a          :            (au)                 -4.915810999714
    6    1    a          :            (au)                 -0.783834121179
    7    1    a          :            (au)                 -0.500541426972
    8    1    a          :            (au)                 -0.188998577200
    9    1    a          :            (au)                 -0.188996563294
   10    1    a          :            (au)                 -0.096572857476
   11    1    b          :            (au)                -76.931062724367
   12    1    b          :            (au)                -13.834606756723
   13    1    b          :            (au)                 -6.781641941526
   14    1    b          :            (au)                 -4.922242495398
   15    1    b          :            (au)                 -4.922239284632
   16    1    b          :            (au)                 -4.915807386492
   17    1    b          :            (au)                 -0.783828308425
   18    1    b          :            (au)                 -0.500540952956
   19    1    b          :            (au)                 -0.188996685089
   20    1    b          :            (au)                 -0.188994770536
   21    1    b          :            (au)                 -0.096571462610
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -228.133101409940
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.690416e+00        -393.644193592948         -3.936442e+02
    1          5.152130e-01        -395.768353364895         -2.124160e+00
    2          2.722755e-01        -395.886814401286         -1.184610e-01
    3          1.185900e-01        -395.904791103832         -1.797670e-02
    4          5.382544e-02        -395.910391727852         -5.600624e-03
    5          4.063945e-02        -395.913643908138         -3.252180e-03
    6          1.396288e-02        -395.914081498684         -4.375905e-04
    7          1.024254e-02        -395.914316424078         -2.349254e-04
    8          7.595514e-03        -395.914374341249         -5.791717e-05
    9          4.108002e-03        -395.914392018967         -1.767772e-05
   10          2.016255e-03        -395.914398234450         -6.215483e-06
   11          1.318488e-03        -395.914400737258         -2.502808e-06
   12          5.473289e-04        -395.914401324040         -5.867824e-07
   13          2.839426e-04        -395.914401473171         -1.491304e-07
   14          1.198984e-04        -395.914401517138         -4.396742e-08
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -0.877386
    1       N            :        0.000000        0.000000        1.940196
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                395.184978874358
 E-N energy              :            (au)              -1015.265253244296
 Coulomb energy          :            (au)                216.746213709837
 Exchange energy         :            (au)                 -7.276441410664
 X-C energy              :            (au)                -22.569902883471
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 37.266003437097
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -433.180404954235
 Nuclear energy          :            (au)                 37.266003437097
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.959144015171e+02
                         :      (kcal/mol)             -2.484400358655e+05
                         :        (kJ/mol)             -1.039473110736e+06
                         :            (eV)             -1.077337922652e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.253798367317
    1    1    a          :            (au)                -14.422504496343
    2    1    a          :            (au)                 -6.656255964971
    3    1    a          :            (au)                 -4.791783020463
    4    1    a          :            (au)                 -4.791783015788
    5    1    a          :            (au)                 -4.786890790810
    6    1    a          :            (au)                 -0.876773803348
    7    1    a          :            (au)                 -0.512676761489
    8    1    a          :            (au)                 -0.349787451384
    9    1    a          :            (au)                 -0.349787440685
   10    1    a          :            (au)                 -0.342433436105
   11    1    b          :            (au)                -77.253798370579
   12    1    b          :            (au)                -14.422504484810
   13    1    b          :            (au)                 -6.656255973324
   14    1    b          :            (au)                 -4.791783024906
   15    1    b          :            (au)                 -4.791783018097
   16    1    b          :            (au)                 -4.786890801579
   17    1    b          :            (au)                 -0.876773828217
   18    1    b          :            (au)                 -0.512676769005
   19    1    b          :            (au)                 -0.349787435168
   20    1    b          :            (au)                 -0.349787424995
   21    1    b          :            (au)                 -0.342433462035
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -230.268949141420
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000        0.000000       -1.553552
 Magnitude               :            (au)                        1.553552
                         :         (Debye)                        3.948734
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.420582
 Magnitude               :            (au)                        0.420582
                         :         (Debye)                        1.069012
---------------------------------------------------------------------------
 Total vector            :       -0.000000        0.000000       -1.132970
 Magnitude               :            (au)                        1.132970
                         :         (Debye)                        2.879722
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 17m  7s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
